exporter/clickhouseexporter/internal/exponential_histogram_metrics.go

```diff
@@ -121,7 +127,7 @@ func (e *expHistogramMetrics) insert(ctx context.Context, db *sql.DB) error {
 		return nil
 	}
 
-	valueArgs := make([]any, e.count*expHistogramValueCounts)
+	valueArgs := make([]any, len(e.expHistogramModels)*e.count*expHistogramValueCounts)
```

Comment:
`e.count` was added by:

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/b2f0015292f57228b8ffbfd2f8fe405ceb6e8de1/exporter/clickhouseexporter/internal/exponential_histogram_metrics.go#L191
so, I think `e.count*expHistogramValueCounts` does calculate the total count. please double check

Reply:
You're right; I didn't see that the value was already accumulated.